src/cf-forms/src/molecules/input-with-btn.less

```diff
@@ -0,0 +1,49 @@
+//
+// Input with button
+//
+
+.m-input-with-btn {
+
+    .respond-to-min( 480px, {
+        .grid_nested-col-group();
+    });
+
+    &_input {
+        margin-bottom: unit( 15px / @base-font-size-px, em );
+        .respond-to-min( 480px, {
+            .grid_column(9);
+            border-right-width: 0;
```

Comment:
I think we can skip it here and discuss it while we figure out layouts. I'd like to collect every form grid scenario on a single page and see what patterns we can discover across them all.